job-descriptions/ux-designer.md

```diff
@@ -0,0 +1,75 @@
+
+If you are passionate about making the world better through software, come join us!
+
+### About the role
+
+We are looking for a self-motivated UX designer with the leadership skills to be the facilitator of design, and the pragmatism to be the person responsible for producing and executing the design. You will be our only designer and will work collaboratively with product and engineering to create a highly functional and beautiful experience that delights a technical audience.
+
+#### Your responsibilities
+
+You will:
+
+- Contribute to the overall strategy and decision-making about Sourcegraph’s features.
+- Drive continual progress towards a cohesive, delightful experience for a technical audience through iterative improvements.
+- Have the autonomy to facilitate conversations, solve problems, and execute the designs.
+- Pull together insights to propose a variety of solutions, then collaborate with product and engineering to determine the best solutions to problems.
+- Use prototyping and usability testing to inform and validate designs.
+
+We will encourage and support you to:
+
+- Collaborate directly with customers to inform strategy and design decisions.
+- Publish blog posts and give conference talks about your work at Sourcegraph.
+
+#### Your qualifications
+
+You have:
+
+- Expertise in designing complex web applications as demonstrated in a strong portfolio showcasing high-quality UX, interaction, and visual design.
+- Proficiency with Figma, Sketch, or similar tools to design end-to-end experiences (UX, interaction, and visual).
+- Experience with usability testing, customer journey mapping, developing personas, wireframing and prototyping, sketching, and other UX design methods.
+- The ability to adapt your tools and processes for the velocity of a fast-paced startup where projects have varying requirement levels.
+- The ability to proactively give maintenance to the design system when thinking about how new concepts impact the whole product.
+- A knack for giving and receiving feedback, and explaining the reasoning behind your designs.
+- Deep understanding of developers, and a good understanding of version control systems, preferably Git.
+
+#### Nice to haves
+
+- You have experience designing developer tools.
+- Experience working in cross-functional teams on multiple projects in tandem.
+- Experience building or contributing to the design of consumer or enterprise applications.
+
+### Location
+
+We are flexible and don't require you to be in any particular timezone or location. Our engineering team is distributed across the world and about half work from our office in San Francisco.
```

Comment:
This specifically mentions our engineering team, which isn't particularly relevant to designers.

```suggestion
We are flexible and don't require you to be in any particular timezone or location. Our team is distributed across the world and less than half work from our office in San Francisco.
```